src/components/List.js

```diff
@@ -1,12 +1,31 @@
-import React from 'react';
+import React, { useContext } from 'react';
+import { SearchResultsContext } from '../context/SearchResults';
+import {
+  getLocationsFromResponseAPI,
+  sortLocationsByDistance,
+} from '../helpers/helpersAPI';
+import { Location } from './Location';
 
-const List = () => (
-  <>
-    <h1>List</h1>
-    <div className="view__content">
-      <ul className="list"></ul>
-    </div>
-  </>
-);
+const List = () => {
+  const responseAPI = useContext(SearchResultsContext);
+  const locationsResponse = getLocationsFromResponseAPI(responseAPI);
+  const locations = sortLocationsByDistance(locationsResponse);
 
+  return (
+    <>
+      <h1 className="list__title">List</h1>
+      <div className="view__content">
+        {locations?.length === 0 ? (
```

Comment:
Love how simple and readable this is! Great work! 👏👏👏

Reply:
Oh, I _was_ going to question this logic, but I see in your helper function you're returning an empty array no matter what. 👍